### PR TITLE
fix todo: add validate method for &schedulerapi.Policy in examples_test.go

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
fix todo: add validate method for &schedulerapi.Policy in examples_test.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hanxiaoshuai** [comment]
cc @hzxuzhonghu

**Comment by thockin** [comment]
@bsalamat has more context on this than I do, even if it is "just" examples.

File: examples/examples_test.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/kubernetes/pkg/registry/batch/job"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
 	schedulerapilatest "k8s.io/kubernetes/pkg/scheduler/api/latest"
+	apivalidation "k8s.io/kubernetes/pkg/scheduler/api/validation"
```
Comment (bsalamat) [inline examples/examples_test.go]: nit: Please rename to `schedulerapivalidation`

**Review by bsalamat — COMMENTED** [review]
Thanks, @hanxiaoshuai! LGTM, just a nit.

File: examples/examples_test.go (hunk 1 above)
Comment (hanxiaoshuai) [reply examples/examples_test.go]: OK,I have done it,Thanks

**Comment by hanxiaoshuai** [comment]
cc @bsalamat

**Comment by hanxiaoshuai** [comment]
cc @bsalamat

File: examples/examples_test.go [hunk 2]
```diff
@@ -144,6 +145,14 @@ func validateObject(obj runtime.Object) (errors field.ErrorList) {
...
 }
 
+func validateschedulerpolicy(obj runtime.Object) error {
+	switch t := obj.(type) {
+	case *schedulerapi.Policy:
+		return schedulerapivalidation.ValidatePolicy(*t)
+	default:
+		return fmt.Errorf("obj type is not schedulerapi.Policy")
+	}
+}
```
Comment (bsalamat) [inline examples/examples_test.go]: Please add a blank line after the function.

Comment (hanxiaoshuai) [reply examples/examples_test.go]: OK, I have re push it.Thanks

**Comment by hanxiaoshuai** [comment]
cc @bsalamat

File: examples/examples_test.go (hunk 1 above)
Comment (brendandburns) [reply examples/examples_test.go]: I don't see this change. Did you forget to push your new changes?

Comment (bsalamat) [reply examples/examples_test.go]: The change was there yesterday, but it has gone back to the first version of the PR.

Comment (hanxiaoshuai) [reply examples/examples_test.go]: cc @brendandburns @bsalamat  I have re pushed it now,Thanks

**Comment by hanxiaoshuai** [comment]
cc @brendandburns

**Pull request merged**